model/src/main/kotlin/OrtResult.kt

```diff
@@ -169,4 +170,7 @@ data class OrtResult(
                     }
                 }
             }
+
+    fun replaceConfig(config: RepositoryConfiguration) =
```

Comment:
Please add a KDoc comment for the function, e.g. "Return a copy of this [OrtResult] with the [Repository.config] replaced by [config]."